src/main/java/org/jabref/gui/groups/GroupTreeController.java

```diff
@@ -52,15 +52,23 @@
 
     private static final Log LOGGER = LogFactory.getLog(GroupTreeController.class);
 
-    @FXML private TreeTableView<GroupNodeViewModel> groupTree;
-    @FXML private TreeTableColumn<GroupNodeViewModel, GroupNodeViewModel> mainColumn;
-    @FXML private TreeTableColumn<GroupNodeViewModel, GroupNodeViewModel> numberColumn;
-    @FXML private TreeTableColumn<GroupNodeViewModel, GroupNodeViewModel> disclosureNodeColumn;
-    @FXML private CustomTextField searchField;
+    @FXML
```

Comment:
I prefer the condensed 1-line form for javafx classes since these @FXML and @Inject tags make the code rather clumsy.... but both forms are fine with me (we should just stick to one convention).

Reply:
This was done by the IntelliJ autoformat function. I'll gladly change it back.